library/template/cache.php

```diff
@@ -94,8 +94,7 @@ public function loadString($content)
      */
     public function render(array $data = array())
     {
-        $this->getDelegate()->render($data);
-        return $this;
+        return $this->getDelegate()->render($data);
```

Comment:
@oligriffiths If you return the result of render() you are jumping out of the decorator, by returning the decorator itself calls can continue to be made against it. Did you consider that ?